check.go

```diff
@@ -75,6 +75,10 @@ type (
 		FirstCheckStartedAt time.Time
 		// ContiguousFails holds the number of how often the check failed in a row.
 		ContiguousFails uint
+		// Ignore will ignore the result/failure of this check when determining the
+		// aggregate final status.
+		Ignore bool
```

Comment:
This was added as a state property, but it never changes. I'd say this is a configuration parameter, so we should probably move it to `checkerConfig`?